src/data_table/local_data_table.js

```diff
@@ -29,6 +29,11 @@ class LocalDataTable extends View {
     _.extend(this, _.pick(this.options, ["selectedIds", "paginate"]));
     _.bindAll(this, "_onRowCreated", "_onBulkHeaderClick", "_onBulkRowClick", "_bulkCheckboxAdjust", "_onDraw",
       "_onColumnVisibilityChange", "_onColumnReorder");
+    if (this.options.urlPagination !== undefined) {
+      this.urlPagination = this.options.urlPagination;
+    } else if (this.urlPagination === undefined) {
+      this.urlPagination = true;
```

Comment:
per other comment I think this would be cleaner and less bug-prone if the else case was:

```
this.urlPagination = this.pagination;
```